packages/amazon-bedrock/src/bedrock-chat-language-model.ts

```diff
@@ -265,11 +274,8 @@ export class BedrockChatLanguageModel implements LanguageModelV1 {
     > = {};
 
     return {
-      stream: stream.pipeThrough(
-        new TransformStream<
-          ParseResult<ConverseStreamOutput>,
-          LanguageModelV1StreamPart
-        >({
+      stream: response.pipeThrough(
+        new TransformStream<ParseResult<any>, LanguageModelV1StreamPart>({
```

Comment:
`any` should be replaced with the actual type from parsing. What is the type of the chunks? we are parsing with Zod?

Reply:
Fixed to reference the chunk schema, added tests for error cases and chunk-parsing-failed that turned up as part of the work.